javascript/features/death_match/death_match_manager.test.js

```diff
@@ -319,4 +310,26 @@ describe('DeathMatchManager', (it, beforeEach) => {
 
         assert.equal(server.textDrawManager.getForPlayer(gunther, textDraw), null);
     });
+
```

Comment:
Why don't we have to test the dialog?

Reply:
This is part of the deathmatch_command. Not the manager.
There's a test clicking on the frist item going to the first zone.